src/In.ProjectEKA.DefaultHip/DataFlow/Collect.cs

```diff
@@ -85,5 +99,18 @@ private List<string> FindPatientData(DataRequest request)
 
             return new List<string>();
         }
+
+        private static void LogDataRequest(DataRequest request)
+        {
+            var ccList = JsonConvert.SerializeObject(request.CareContexts);
+            var requestedHiTypes = string.Join(", ", request.HiType.Select(hiType => hiType.ToString()));
+            var transactionId = request.TransactionId;
+            var from = request.DataRange.From;
+            var to = request.DataRange.To;
+            var callbackUrl = request.CallBackUrl;
+            Log.Information(
```

Comment:
Do we need to define these? can't we use them directly?

Reply:
Thats what I thought. But it was throwing compilation errors. However, the documentation said such formatting should be fine.

Reply:
Not all of them, only ```var transactionId = request.TransactionId;
            var from = request.DataRange.From;
            var to = request.DataRange.To;
            var callbackUrl = request.CallBackUrl```